refgram/morphology/index.html

```diff
@@ -284,7 +284,7 @@ <h3 id="passivizer_prefixes"><a href="#passivizer_prefixes">Passivizer prefixes<
 		<div class="vocabularyt">te-</div>
 	</td>
 	<td>
-		<div class="vocabularyd">perfect participle (<span class="sc">ppf</span>)</div>
+		<div class="vocabularyd">passive perfect participle (<span class="sc">ppf</span>)</div>
```

Comment:
```suggestion
		<div class="vocabularyd">perfect passive participle (<span class="sc">ppf</span>)</div>
```

I'm used to seeing it written this way